CHANGELOG.md

```diff
@@ -21,6 +21,7 @@ a release.
 ## [Unreleased]
 
 ### Fixed
+- Timestampable: Type handling for the tracked field values configured in the timestampable field.
```

Comment:
this also fixes it in `Blameable` and `IpTraceable` extensions which use `AbstractTracking`.

Reply:
Good catch.